src/modules/edit.js

```diff
@@ -1,39 +1,11 @@
-export const savedTasks = JSON.parse(localStorage.getItem('tasks')) || [];
+import { saveTasks, savedTasks } from './localStorage.js';
 
-export function saveTasks() {
-  localStorage.setItem('tasks', JSON.stringify(savedTasks));
-}
-
-export function addTask(taskValue) {
-  if (taskValue === '') {
-    return;
-  }
-
-  const task = {
-    text: taskValue,
-    completed: false,
-    index: savedTasks.length + 1,
-  };
-
-  savedTasks.push(task);
+export function updateStatus(index, completed) {
+  savedTasks[index].completed = completed;
   saveTasks();
 }
```

Comment:
- [ ] dear @SAZZAD1Q2, Here for the completed methods we need to have a new JavaScript file imported as a module. This is to make sure you understand importing modules in your project. Please implement the functionality for status updates (`completed`: `true` / `false`) in a new file and import it as a module. 
- [ ] Also, when the user clicks on the check box, we need to make sure the task status is changed, Also the change should be stored in the local storage. This makes your app a lot more flexible.